content/en/blog/2023/opentelemetry-in-focus-january-2023.md

```diff
@@ -0,0 +1,85 @@
+---
```

Comment:
Consider renaming the file to: `content/en/blog/2023/otel-in-focus-2023-01.md` (we could even drop the `2023-` and just use the name `2023/otel-in-focus-01.md`)